examples/ITK_Example15_Transformix_image_TranslationTransform.ipynb

```diff
@@ -0,0 +1,174 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "9fbb7de3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Moving image:\n",
+      "[[ 1.  2.  3.  4.  5.]\n",
+      " [ 6.  7.  8.  9. 10.]\n",
+      " [11. 12. 13. 14. 15.]\n",
+      " [16. 17. 18. 19. 20.]\n",
+      " [21. 22. 23. 24. 25.]\n",
+      " [26. 27. 28. 29. 30.]]\n",
+      "\n",
+      "Translation: [1, -2]\n"
+     ]
+    }
+   ],
+   "source": [
+    "number_of_columns = 5\n",
+    "number_of_rows = 6\n",
+    "\n",
+    "moving_image = ImageType.New()\n",
+    "moving_image.SetRegions([number_of_columns, number_of_rows])\n",
+    "moving_image.Allocate(True)\n",
+    "\n",
+    "# Set the pixel values consecutively to 1, 2, 3, ..., n.\n",
+    "moving_image[:] =  np.arange(1, number_of_columns*number_of_rows + 1).reshape((number_of_rows, number_of_columns))\n",
+    "                            \n",
+    "print('Moving image:')\n",
+    "print(np.asarray(moving_image))\n",
+    "print()\n",
+    "\n",
+    "translation = [1, -2]\n",
+    "print('Translation:', translation)\n",
+    "\n",
+    "transform = itk.TranslationTransform.New()\n",
+    "transform.SetOffset(translation)\n",
+    "\n",
+    "parameter_map = {\n",
+    "                 \"Direction\": (\"1\", \"0\", \"0\", \"1\"),\n",
+    "                 \"Index\": (\"0\", \"0\"),\n",
+    "                 \"Origin\": (\"0\", \"0\"),\n",
+    "                 \"Size\": (str(number_of_columns), str(number_of_rows)),\n",
+    "                 \"Spacing\": (\"1\", \"1\")\n",
+    "                }"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2947cf4a",
+   "metadata": {},
+   "source": [
+    "Pass the example input to the TransformixFilter object, apply the transformation, and retrieve the output image. Ready!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "26f9394e",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Output image:\n",
+      "[[ 0.  0.  0.  0.  0.]\n",
+      " [ 0.  0.  0.  0.  0.]\n",
+      " [ 2.  3.  4.  5.  0.]\n",
+      " [ 7.  8.  9. 10.  0.]\n",
+      " [12. 13. 14. 15.  0.]\n",
+      " [17. 18. 19. 20.  0.]]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "parameter_object = itk.ParameterObject.New()\n",
+    "parameter_object.AddParameterMap(parameter_map)\n",
+    "\n",
+    "transformix_filter.SetMovingImage(moving_image)\n",
+    "transformix_filter.SetTransformParameterObject(parameter_object)\n",
+    "transformix_filter.SetTransform(transform)\n",
+    "transformix_filter.Update()\n",
+    "\n",
+    "output_image = transformix_filter.GetOutput()\n",
+    "\n",
+    "print('Output image:')\n",
+    "print(np.asarray(output_image))\n",
+    "print()"
```

Comment:
Where is this print command for?

Reply:
`print()` just prints a line-break. But I guess I could have done `print(np.asarray(output_image), "\n")`, instead of line 148-149.